gs_quant/entities/entity.py

```diff
@@ -111,6 +111,7 @@ def get(cls,
             id_type: Union[EntityIdentifier, str],
             entity_type: Optional[Union[EntityType, str]] = None):
         id_type = id_type.value if isinstance(id_type, Enum) else id_type
+        id_value = id_value.lower()  # Convert to lowercase
```

Comment:
Suggestion: Ensure that id_value is a string before calling .lower() to avoid potential errors if id_value is None or not a string.

```
if isinstance(id_value, str):
  id_value = id_value.lower()

```